app/src/ui/repositories-list/repositories-list.tsx

```diff
@@ -161,9 +160,8 @@ export class RepositoriesList extends React.Component<
   private renderGroupHeader = (id: string) => {
     const identifier = id as RepositoryGroupIdentifier
     const label = this.getGroupLabel(identifier)
-    const className = enableGroupRepositoriesByOwner()
-      ? 'filter-list-group-header group-repositories-by-owner'
-      : 'filter-list-group-header'
+    const className = 'filter-list-group-header group-repositories-by-owner'
```

Comment:
I think we should take this opportunity to remove the `group-repositories-by-owner` meta class and roll up those rules into the `filter-list-group-group-header` class since that's always going to be applied now. If you want to tackle that in a separate PR that's fine but I think that a part of removing the feature flags should be removing these feature flag specific css classes.

Reply:
I think I prefer to tackle this specific cleanup in a separate PR, since I'm not sure this cleanup can be as trivial as just removing a few CSS style declarations (maybe I'm wrong)

Reply:
Ok the cleanup turned up much easier than I anticipated 😜 so I've ended up including it in this PR as 844c10bab1802f862ffc9bc6d2667fcc334d876a